packages/netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl.js

```diff
@@ -327,9 +331,11 @@ class EditorControl extends React.Component {
               isFieldHidden={isFieldHidden}
             />
             {fieldHint && (
-              <ControlHint active={isSelected || this.state.styleActive} error={hasErrors}>
-                {fieldHint}
-              </ControlHint>
+              <ControlHint
```

Comment:
I believe the intention in the issue is to support markdown, not injecting HTML directly (which we would like to avoid).

We could use https://github.com/remarkjs/react-markdown or even [expose the one we're using for the markdown widget preview](https://github.com/netlify/netlify-cms/blob/574025fd649d49f70650a6112617642a6cb24f07/packages/netlify-cms-widget-markdown/src/MarkdownPreview.js#L20)

Reply:
Ah cool - sounds good! I will update to use one or the other with markdown.